tests/shared/audit/30-ospp-v42-4-delete-failed.rules

```diff
@@ -1,5 +1,5 @@
 ## Unsuccessful file delete
--a always,exit -F arch=b32 -S unlink,unlinkat,rename,renameat -F exit=-EACCES -F auid>=1000 -F auid!=unset -F key=unsuccessful-delete
```

Comment:
Please remove modifications to this file, it is used for testing of a different rule and it does not influence the rule modified by this PR.

Reply:
Done